db/migrations/004_match_game_count.rb

```diff
@@ -0,0 +1,18 @@
+Sequel.migration do
+  up do
+    alter_table(:users_seasons_matches) do
+      add_column :game_wins, Integer, :null => true, :default => 0
+    end
+    alter_table(:matches) do
+      add_column :best_of, Integer, :null => true, :default => 3
+    end
+  end
+  down do
+    alter_table(:users_seasons_matches) do
+      drop_column :game_wins
+    end
+    alter_table(:matches) do
```

Comment:
I have the ui for adding a `best_of` field to a match creation. I don't see that here so I can just commit mine.